doc/install/install.rst

```diff
@@ -1,296 +1,147 @@
- - `core_diversity_analyses.py <../scripts/core_diversity_analyses.html>`_
+::
 
-Prepare your build environment...
----------------------------------
+    conda remove --name qiime191 --all
 
-First, you need to have a build environment on your machine:
+These instructions have been tested on Mac OS X (10.8–10.11) and Linux
+(Ubuntu 14.04 and Mint 17.3 Rosa).
 
- - On OS X, this involves installing the `developer tools <http://developer.apple.com/technologies/xcode.html>`_.
- - On Debian-based Linux (e.g., Ubuntu), this involves installing the following packages::
+Installing QIIME dependencies in your Conda environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-     sudo apt-get install build-essential python-dev python-pip
+You now have the base installation and access to much of QIIME's
+functionality. However, if you want to use additional dependencies, your
+Conda installation of QIIME will need to have access to the executable
+files for each program you'd like to invoke as a dependency.
+Commonly-used QIIME dependencies include UCLUST (now included in the
+base installation), RDP Classifier, BLAST, USEARCH, and VSEARCH.
 
-.. warning:: Depending on your system, additional packages may need to be installed that may be required by QIIME or its dependencies. Also, packages names will vary across operating systems and even within operating system distributions (e.g., different versions of Ubuntu). Since there are a wide range of systems that QIIME can be installed on, the QIIME development team cannot provide a comprehensive list of commands that are guaranteed to successfully install QIIME on your system.
+Linking to executables you already have
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-QIIME is written in Python 2, and requires Python version greater than Python 2.7.0 and less than Python 3.0.0. You can verify which version of Python you're running with::
+If you already have the appropriate version of these programs installed,
+you can ensure that they run the correct executable by making a symbolic
+link in the bin folder of your Conda QIIME environment. To allow your
+environment to access dependency executables, you can put symbolic links
+to them in your environment's bin folder. The path to this folder is
+displayed when you load the environment:
 
-    python --version
+::
 
+    source activate qiime191
+    discarding /home/username/miniconda3/bin from PATH
+    prepending /home/username/miniconda3/envs/qiime191/bin to PATH
 
-If you encounter problems installing QIIME, please visit http://help.qiime.org.
+To make a symbolic link:
 
-Then, just pip install qiime!
------------------------------
+::
 
-The easiest way to install the latest QIIME release and its base dependencies is with pip::
+    ln -s /path/to/myexecutable /home/username/miniconda3/envs/qiime191/bin/myexecutable
 
-	pip install numpy
-	pip install qiime
+Downloading and linking to new executables
```

Comment:
This seems repetitive with the previous section. Could we just have a single "Using other executables" section? Fewer instructions for us to maintain/support. Or, if they just work if they're in your ``$PATH``, maybe we can drop this discussion all together?